src/clr-addons/content-panel/content-panel.ts

```diff
@@ -53,10 +53,10 @@ export class ClrContentPanel implements OnInit {
     this._open = false;
     this.closed.emit();
     // this event is used by the tree table to adjust the colomns width
-    // 220 is for the animation to finish.
+    // 200 is for the animation to finish + 60 safety marge.
     setTimeout(() => {
       window.dispatchEvent(new Event('resize'));
-    }, 220);
+    }, 260);
```

Comment:
Can we use Animation callbacks for these? As timeouts will work on some clients, but on slower clients won't.
https://angular.io/guide/transition-and-triggers#animation-callbacks

Reply:
sounds a good idea, I will try it

Reply:
is 5.0.1-alpha-1 fine ?